opencti-platform/opencti-graphql/src/domain/container.js

```diff
@@ -213,3 +223,18 @@ export const knowledgeAddFromInvestigation = async (context, user, { containerId
   // Reload on this is mandatory to get the rel_ from the element for accurate counting
   return internalLoadById(context, user, patched.internal_id, patched.entity_type);
 };
+
+export const containerEditAuthorizedMembers = async (context, user, entityId, input) => {
+  if (!isFeatureEnabled('CONTAINERS_AUTHORIZED_MEMBERS')) {
```

Comment:
we should test if it's a feedback too, since feedback are not part of feature flag, or remove this feature flag test here